app/controllers/api/projects_controller.rb

```diff
@@ -101,11 +107,14 @@ def project_params
     end
 
     def project_scope
-      @project_scope = project_source.viewable_by(current_user).
-        joins(:data_source, :organization).
-        where(computed_project_type: @project_types).
-        merge(data_source_source.where(id: @data_source_ids)).
-        merge(organization_source.where(id: @organization_ids))
+      @project_scope ||= begin
```

Comment:
`project_types`, 'data_source_ids`, and, `organization_ids` don't memoize, but this does... Since, I think it only gets used in the controller cycle, I think this is fine, but maybe make it `private`?

Reply:
The others memoize, but only after the initial check.